application-model/src/main/java/org/bonitasoft/engine/business/application/xml/AdvancedApplicationNode.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2019 Bonitasoft S.A.
+ * Bonitasoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ **/
+package org.bonitasoft.engine.business.application.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+
+/**
+ * @author Elias Ricken de Medeiros
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+public class AdvancedApplicationNode extends AbstractApplicationNode {
+
+    @Override
+    public boolean equals(Object o) {
+        // super is enough
+        return super.equals(o);
+    }
+
+    @Override
+    public int hashCode() {
+        // super is enough
+        return super.hashCode();
```

Comment:
Might not be necessary to override equals & hashCode if they only call super methods